src/main/java/com/jalasoft/devfund2/controller/HelloController.java

```diff
@@ -14,4 +20,21 @@ public String sayHello(@RequestParam(value="name") String name,
                            @RequestParam(value="lastName") String lname) {
         return ("Hello Work " + name + " " + lname);
     }
+
+    @PostMapping("/upload")
+    public String uploadFile(@RequestParam(value="file") MultipartFile file) throws IOException {
```

Comment:
in this point we need to use try catch and send the error to the client, we will talk about that in the class. anyway good job.